### PR TITLE
Update GKE getting started docs.

### PR DESCRIPTION
**Pull request by IMBurbank**
Fixes [kubeflow/kubeflow #1898](https://github.com/kubeflow/kubeflow/issues/1898)

Updated [GKE quickstart](https://www.kubeflow.org/docs/started/getting-started-gke/) and [troubleshooting](https://www.kubeflow.org/docs/guides/gke/troubleshooting-gke/) docs to address the outstanding issues in [kubeflow/kubeflow #1898](https://github.com/kubeflow/kubeflow/issues/1898).

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/326)

### CODE REVIEW

File: content/docs/started/getting-started-gke.md [hunk 1]
```diff
@@ -56,7 +69,7 @@ Create an OAuth client ID to be used to identify Cloud IAP when requesting acces
         ```
         * `<name>` and `<project>` must have the same values as set in the next
           step when you run the deployment script.
-        * The deployment uses "kubeflow" by default for `<name>` but you can
+        * The deployment uses "ks-kubeflow" by default for `<name>` but you can
```
Comment (abhi-g) [inline content/docs/started/getting-started-gke.md]: Is that true? I believe the default is "kubeflow"

Comment (IMBurbank) [reply content/docs/started/getting-started-gke.md]: Let me correct this. The correct answer is that it defaults to the `KFAPP` value.

Comment (IMBurbank) [reply content/docs/started/getting-started-gke.md]: I've clarified the relationship between `<name>`, `KFAPP`, and `DEPLOYMENT_NAME`. Namely that when `kftcl.sh init $KFAPP ...` is run, `DEPLOYMENT_NAME` is assigned to the `KFAPP` value, saved in `${KFAPP}/env.sh` and subsequently used as the `<name>` value for the envoy-ingress endpoint.

**Pull request merged**